tensorflow/core/kernels/image/resize_bicubic_op_test.cc

```diff
@@ -92,7 +92,13 @@ class ResizeBicubicOpTest : public OpsTestBase {
                                    std::array<float, 4>* weights,
                                    std::array<int64_t, 4>* indices) {
     const int64_t in_loc = scale * out_loc;
-    const float delta = scale * out_loc - in_loc;
+    // Ensure that the following calculation is kept in a float to match the
```

Comment:
Why not use multi line comment?

Reply:
Google style guide https://google.github.io/styleguide/cppguide.html says to be consistent and // is used elsewhere in this file even for multi-line.